src/dmd/errors.d

```diff
@@ -233,6 +233,31 @@ private void verrorPrint(const ref Loc loc, Color headerColor, const(char)* head
     else
         fputs(tmp.peekString(), stderr);
     fputc('\n', stderr);
+
+    if (global.params.printErrorContext &&
+        // ignore invalid files
+        loc != Loc.initial &&
+        // ignore mixins for now
+        !loc.filename.strstr(".d-mixin-"))
```

Comment:
Since the addition of `-mixin=<filename>` might want to doublecheck that logic.

Reply:
Good point. For now, I just added `&& !global.params.mixinOut` as that's definitely safe.
In the future, we probably need to call `flushMixins` or make the fileCache aware of mixin files.